### PR TITLE
[RHPAM-3460] - LDAP login module flag for Kie Server is not set in Op…

### PR DESCRIPTION
**Pull request by msmagnanijr**
…erator

See: https://issues.redhat.com/browse/RHPAM-3460

Signed-off-by: Mauricio Magnani <msmagnanijr@gmail.com>

### CODE REVIEW

**Comment by msmagnanijr** [comment]
@spolti could you please take a look? thanks.

**Review by spolti — APPROVED** [review]

File: config/7.10.1/common.yaml [hunk 1]
```diff
@@ -727,6 +727,8 @@ servers:
                       value: "[[$.Auth.LDAP.BindCredential]]"
                     - name: AUTH_LDAP_JAAS_SECURITY_DOMAIN
                       value: "[[$.Auth.LDAP.JAASSecurityDomain]]"
+                    - name: AUTH_LDAP_LOGIN_MODULE
+                      value: "[[.Auth.LDAP.LoginModule]]"
```
Comment (tchughesiv) [inline config/7.10.1/common.yaml:731]: appears you're missing `$` on front of env var as the others have set

**Review by tchughesiv — CHANGES_REQUESTED** [review]

File: config/7.10.1/common.yaml (hunk 1 above)
Comment (msmagnanijr) [reply config/7.10.1/common.yaml]: thanks!

**Pull request merged**